tutorials/poststack_nccl.py

```diff
@@ -0,0 +1,197 @@
+r"""
+Post Stack Inversion - 3D with Nvidia's NCCL
+=========================
+This tutorial is similar to poststack.py but it shows how to run PyLops-MPI in multi-GPU environment
```

Comment:
Instead of `poststack.py` see this to add an hyperlink to the other poststack tutorial https://github.com/sphinx-gallery/sphinx-gallery/issues/805